ssc/cmod_pvsamv1.cpp

```diff
@@ -539,6 +539,12 @@ static var_info _cm_vtab_pvsamv1[] = {
         { SSC_OUTPUT,        SSC_ARRAY,      "df",                                         "Irradiance DHI from weather file",                                     "W/m2",   "",                      "Time Series",       "*",                    "",                              "" },
         { SSC_OUTPUT,        SSC_ARRAY,      "wfpoa",                                      "Irradiance POA from weather file",                                     "W/m2",   "",                      "Time Series",       "",                     "",                              "" },
 
+        // header data from weather file
+        { SSC_OUTPUT,        SSC_NUMBER,      "lat",                                         "Latitude from weather file",                                          "DD",   "",                      "Location",       "",                    "",                              "" },
```

Comment:
Is DD understood by enough of the modeling community to use as a abbreviated unit (probably yes)

Reply:
I'm not sure how well known it is among the solar modeling community, but I've been using it based on information like this [Wikipedia article](https://en.wikipedia.org/wiki/Decimal_degrees) since there has been confusion in the past about geographic coordinates in SAM, such as using both a negative sign and a "W" for longitude values for locations in the United States. I can change the units to "deg" or "degrees" if that seems clearer than "DD". (The Location and Resource input page uses "DD".)

Reply:
I didn't notice that we have been using DD, I would say keep as you've implemented it.

Reply:
Hm. there is this PV Terms project that uses degrees for latitude longitude: https://duramat.github.io/pv-terms/

> latitude [degrees]: Latitude between -90 and 90 degrees. Positive north of the equator.
> longitude [degrees]: Longitude between -180 to 180 degrees. Positive east of the meridian.